README.md

```diff
@@ -1,34 +1,27 @@
-This README is just a fast *quick start* document. You can find more detailed documentation at [redis.io](https://redis.io).
+This README is just a fast *quick start* document. You can find more detailed documentation at [valkey.io](https://valkey.io/)
 
-What is Redis?
---------------
-
-Redis is often referred to as a *data structures* server. What this means is that Redis provides access to mutable data structures via a set of commands, which are sent using a *server-client* model with TCP sockets and a simple protocol. So different processes can query and modify the same data structures in a shared way.
-
-Data structures implemented into Redis have a few special properties:
+What is Valkey?
+---------------
 
-* Redis cares to store them on disk, even if they are always served and modified into the server memory. This means that Redis is fast, but that it is also non-volatile.
-* The implementation of data structures emphasizes memory efficiency, so data structures inside Redis will likely use less memory compared to the same data structure modelled using a high-level programming language.
-* Redis offers a number of features that are natural to find in a database, like replication, tunable levels of durability, clustering, and high availability.
+Valkey is an open source fork of Redis OSS, created at the point where Redis switched to a proprietary license.
```

Comment:
```suggestion
Valkey is an open source fork of Redis OSS, created at the point where Redis switched to a source-available license.
```
Some people nitpick about this.

Reply:
Then I will nit-pick that source-available doesn't imply that it's non-open-source, which is want the text to say.

Reply:
Then, "switched away form an open-source license" is precise.